charts/workflow/values.yaml

```diff
@@ -107,6 +113,10 @@ controller:
   # disabled - turns off open registration
   # admin_only - allows for registration by an admin only.
   registration_mode: "admin_only"
+  # The public resolvable hostname to build your cluster with.
```

Comment:
Changed to "_publicly resolvable_" to keep it simple

If we want to elaborate more on this later we can.